configsuite/config.py

```diff
@@ -26,6 +26,13 @@
 from .meta_keys import MetaKeys as MK
 
 
+_RETURN_TYPE = {
```

Comment:
Not the best solution this one, open for other suggestions..

Considered including the empty container as part of the `Type` - `collections.NamedTuple`, but didn't really fan out